serach.py

```diff
@@ -0,0 +1,22 @@
+import uuid
+import json
+import milvus_model
+import sys, getopt
+from pymilvus import MilvusClient, DataType, Collection
+
+CLUSTER_ENDPOINT="http://localhost:27017"
+client = MilvusClient(uri=CLUSTER_ENDPOINT)
+
+client.load_collection("ceph_bkt_3cab575e_5080_4e5c_9675_86b8f53af5c7")
+
+embedding_fn = milvus_model.dense.SentenceTransformerEmbeddingFunction(model_name='all-MiniLM-L6-v2',device='cpu')
```

Comment:
this is different than the embedding here:
https://github.com/thotz/python-vectordbapp-ceph/pull/1/files#diff-a56338178149e9f454de84ee47b5110e8a84f1cdd7019591d6eb07508ddbd641R66

will that work?

Reply:
I changed embedding function to reduce the size of container image to test the image usecase for the timebeing